server/server.go

```diff
@@ -22,6 +26,10 @@ const (
 type Server struct {
 	// port is the port number that an HTTP server will listen for requests on.
 	port int
+
+	// activeGames stores pointers to Interactors for all ongoing games that the
+	// server is handling.
```

Comment:
```suggestion
	// server is handling, indexed by gameIDs.
```

The doc comment on the `Interactor.Players` field explains what the keys of that map are. Will you include an explanation of what this map's keys are?